src/Microsoft.ML.Data/Commands/TrainCommand.cs

```diff
@@ -70,9 +70,6 @@ public sealed class Arguments : DataCommand.ArgumentsBase
             [Argument(ArgumentType.AtMostOnce, IsInputFileName = true, HelpText = "The test data file", ShortName = "test")]
             public string TestFile;
 
-            [Argument(ArgumentType.LastOccurenceWins, HelpText = "Whether we should cache input training data", ShortName = "cache")]
-            public bool? CacheData;
```

Comment:
I think removing the `Transforms.DataCache` entry point is fine, but I don't think we want to remove this option from our training commands, do we?

Reply:
The commands are self contained applications. It's just fine for them to cache.

To be clear, what I was talking about was removing the *cache entry point specifically*, and that is all. I was not suggesting anything broader or more ambitious than this. We *still want this command to have an option to cache*.